src/framework/core/js/FluidDocument.js

```diff
@@ -174,4 +174,17 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         };
     });
 
+    /* Sets the value to the DOM element and triggers the change event on the element.
+     * Note: when using jQuery val() function to change the node value, the change event would
+     * not be fired automatically, it requires to be initiated by the user.
+     *
+     * @param node {A jQueryable DOM element} A selector, a DOM node, or a jQuery instance
+     * @param value {String|Number|Array} A string of text, a number, or an array of strings
+     * corresponding to the value of each matched element to set in the node
+     */
+    fluid.changeElementValue = function (node, value) {
```

Comment:
@amb26, the use case that triggers the extraction of changeElementValue() (https://github.com/GPII/first-discovery/pull/47#discussion-diff-30366584) uses this function in the code that implements actual functionalities instead of code for testing, so I wonder if it makes sense to have this function in the core framework for a broader use. 

What do you think, @amb26?